2017-07-28 07:58:22.264 [error] <0.2155.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.2160.0> exit with reason normal in context child_terminated
2017-07-28 08:35:03.297 [error] <0.2148.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.2153.0> exit with reason normal in context child_terminated
2017-07-28 08:39:41.503 [error] <0.187.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.192.0> exit with reason normal in context child_terminated
2017-07-28 08:41:39.703 [error] <0.187.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.192.0> exit with reason normal in context child_terminated
2017-07-28 08:42:47.638 [error] <0.273.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.278.0> exit with reason normal in context child_terminated
2017-07-28 09:02:08.159 [error] <0.273.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.278.0> exit with reason normal in context child_terminated
2017-07-28 09:20:35.303 [error] <0.201.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.206.0> exit with reason normal in context child_terminated
2017-07-28 09:26:03.545 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-28 09:41:00.789 [error] <0.178.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.183.0> exit with reason normal in context child_terminated
2017-07-28 09:41:10.690 [error] <0.178.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.183.0> exit with reason normal in context child_terminated
2017-07-28 10:29:31.081 [error] <0.187.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.192.0> exit with reason normal in context child_terminated
2017-07-28 10:55:15.734 [error] <0.273.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.278.0> exit with reason normal in context child_terminated
2017-07-28 10:55:22.172 [error] <0.187.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.192.0> exit with reason normal in context child_terminated
2017-07-28 12:03:13.908 [error] <0.2150.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.2155.0> exit with reason normal in context child_terminated
2017-07-28 12:41:00.196 [error] <0.231.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.236.0> exit with reason normal in context child_terminated
2017-07-28 12:46:41.905 [error] <0.417.0> CRASH REPORT Process <0.417.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 12:46:41.923 [error] <0.416.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.416.0>, stream 1 had its request process <0.417.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},50832},pid => <0.416.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 12:56:29.593 [error] <0.231.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.236.0> exit with reason normal in context child_terminated
2017-07-28 12:57:03.230 [error] <0.413.0> CRASH REPORT Process <0.413.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 12:57:03.248 [error] <0.412.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.412.0>, stream 1 had its request process <0.413.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},50946},pid => <0.412.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:14:25.535 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-28 13:14:48.152 [error] <0.377.0> CRASH REPORT Process <0.377.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 13:14:48.166 [error] <0.376.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.376.0>, stream 1 had its request process <0.377.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51000},pid => <0.376.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:20:47.513 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-28 13:20:56.048 [error] <0.377.0> CRASH REPORT Process <0.377.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 13:20:56.071 [error] <0.376.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.376.0>, stream 1 had its request process <0.377.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51136},pid => <0.376.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:25:46.909 [error] <0.227.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.232.0> exit with reason normal in context child_terminated
2017-07-28 13:26:22.245 [error] <0.409.0> CRASH REPORT Process <0.409.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 13:26:22.265 [error] <0.408.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.408.0>, stream 1 had its request process <0.409.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51156},pid => <0.408.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:29:57.562 [error] <0.413.0> CRASH REPORT Process <0.413.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 13:29:57.563 [error] <0.412.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.412.0>, stream 1 had its request process <0.413.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51170},pid => <0.412.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:37:19.309 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-28 13:37:29.565 [error] <0.366.0> CRASH REPORT Process <0.366.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 13:37:29.585 [error] <0.365.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.365.0>, stream 1 had its request process <0.366.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51248},pid => <0.365.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:49:34.313 [error] <0.374.0> CRASH REPORT Process <0.374.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 13:49:34.314 [error] <0.373.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.373.0>, stream 1 had its request process <0.374.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51324},pid => <0.373.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 13:56:16.270 [error] <0.257.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.262.0> exit with reason normal in context child_terminated
2017-07-28 14:00:51.480 [error] <0.257.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.262.0> exit with reason normal in context child_terminated
2017-07-28 14:00:55.032 [error] <0.439.0> CRASH REPORT Process <0.439.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:00:55.043 [error] <0.438.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.438.0>, stream 1 had its request process <0.439.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51420},pid => <0.438.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:01:52.434 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-28 14:02:19.285 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-28 14:02:39.887 [error] <0.366.0> CRASH REPORT Process <0.366.0> with 0 neighbours exited with reason: call to undefined function 'Elixir.Question.Question.Server':'__call_rpc__'(<<"/QuestionService/GetAllQuestions">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:02:39.908 [error] <0.365.0> Ranch listener 'Elixir.Question.Question.Server', connection process <0.365.0>, stream 1 had its request process <0.366.0> exit with reason undef and stacktrace [{'Elixir.Question.Question.Server','__call_rpc__',[<<"/QuestionService/GetAllQuestions">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/GetAllQuestions">>,path_info => undefined,peer => {{172,17,0,1},51442},pid => <0.365.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.Question.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.Question.Server',unmarshal => nil}],[]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:14:40.890 [error] <0.257.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.262.0> exit with reason normal in context child_terminated
2017-07-28 14:18:32.341 [error] <0.257.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.262.0> exit with reason normal in context child_terminated
2017-07-28 14:22:24.490 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-28 14:22:34.255 [error] <0.377.0> CRASH REPORT Process <0.377.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:22:34.277 [error] <0.376.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.376.0>, stream 1 had its request process <0.377.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},51532},pid => <0.376.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:30:28.507 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-28 14:30:48.438 [error] <0.377.0> CRASH REPORT Process <0.377.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:30:48.459 [error] <0.376.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.376.0>, stream 1 had its request process <0.377.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},51738},pid => <0.376.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:41:35.419 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-28 14:43:26.071 [error] <0.379.0> CRASH REPORT Process <0.379.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:43:26.086 [error] <0.378.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.378.0>, stream 1 had its request process <0.379.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},51988},pid => <0.378.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:44:24.221 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-28 14:44:57.066 [error] <0.367.0> CRASH REPORT Process <0.367.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:44:57.087 [error] <0.366.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.366.0>, stream 1 had its request process <0.367.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},52004},pid => <0.366.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:45:26.389 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-28 14:48:36.773 [error] <0.367.0> CRASH REPORT Process <0.367.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:48:36.788 [error] <0.365.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.365.0>, stream 1 had its request process <0.367.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},52022},pid => <0.365.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 14:55:18.299 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-28 14:56:53.077 [error] <0.366.0> CRASH REPORT Process <0.366.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 14:56:53.098 [error] <0.365.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.365.0>, stream 1 had its request process <0.366.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},52366},pid => <0.365.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-28 15:08:49.695 [error] <0.257.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.262.0> exit with reason normal in context child_terminated
2017-07-28 15:09:04.891 [error] <0.439.0> CRASH REPORT Process <0.439.0> with 0 neighbours exited with reason: no function clause matching 'Elixir.Question.QuestionService.Server':'__call_rpc__'(<<"/QuestionService/QuestionCall">>, #{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,...}) line 4 in 'Elixir.GRPC.Adapter.Cowboy.StreamHandler':proc_lib_hack/3 line 144
2017-07-28 15:09:04.899 [error] <0.438.0> Ranch listener 'Elixir.Question.QuestionService.Server', connection process <0.438.0>, stream 1 had its request process <0.439.0> exit with reason function_clause and stacktrace [{'Elixir.Question.QuestionService.Server','__call_rpc__',[<<"/QuestionService/QuestionCall">>,#{'__struct__' => 'Elixir.GRPC.Server.Stream',adapter => 'Elixir.GRPC.Adapter.Cowboy',marshal => nil,payload => #{bindings => #{},body_length => undefined,has_body => true,has_sent_resp => headers,headers => #{<<"content-type">> => <<"application/grpc+proto">>,<<"te">> => <<"trailers">>,<<"user-agent">> => <<"grpc-elixir/0.1.0">>},host => <<"localhost">>,host_info => undefined,method => <<"POST">>,path => <<"/QuestionService/QuestionCall">>,path_info => undefined,peer => {{172,17,0,1},52492},pid => <0.438.0>,port => undefined,qs => <<>>,ref => 'Elixir.Question.QuestionService.Server',scheme => <<"http">>,streamid => 1,version => 'HTTP/2'},server => 'Elixir.Question.QuestionService.Server',unmarshal => nil}],[{file,"lib/question_server.ex"},{line,4}]},{'Elixir.GRPC.Adapter.Cowboy.Handler',init,2,[{file,"lib/grpc/adapter/cowboy/handler.ex"},{line,17}]},{cowboy_handler,execute,2,[{file,"/home/nemanja/workspace/question_server/deps/cowboy/src/cowboy_handler.erl"},{line,37}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',execute,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,152}]},{'Elixir.GRPC.Adapter.Cowboy.StreamHandler',proc_lib_hack,3,[{file,"lib/grpc/adapter/cowboy/stream_handler.ex"},{line,139}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]
2017-07-30 20:33:10.393 [error] <0.2174.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.2179.0> exit with reason normal in context child_terminated
2017-07-30 20:44:31.425 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-30 20:47:58.634 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-30 20:58:40.675 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-30 20:59:59.425 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
2017-07-30 21:01:40.147 [error] <0.184.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.189.0> exit with reason normal in context child_terminated
2017-07-30 21:04:06.087 [error] <0.195.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':watcher(error_logger, 'Elixir.Logger.ErrorHandler', {true,false,500}) at <0.200.0> exit with reason normal in context child_terminated
